msa-persistence/src/main/java/cz/muni/fi/pa165/monsterslayeragency/entities/Request.java

```diff
@@ -33,7 +33,6 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @Column(nullable = false)
```

Comment:
As a request must come from someone I would consider keeping this here.

Reply:
An attribute with many-to-one annotation should not have an additional column annotation.